gtk/src/default/gtk-3.20/_common.scss

```diff
@@ -2841,7 +2849,7 @@ switch {
     &:checked {
       @if $variant == 'light' { color: $backdrop_bg_color; }
       border-color: if($variant == 'light', $checkradio_borders_color,
-                                            $checkradio_borders_color);
+                                            $selected_borders_color);
```

Comment:
do we lose our check/radio color customization?

Reply:
Same as above